docs/managing-minishift.md

```diff
@@ -181,8 +183,20 @@ These are not configurable at the moment and are different for each driver and t
 <a name="mounting-custom-shared-folders"></a>
 ### Mounting custom shared folders
 
-The Boot2Docker and the CentOS image include `cifs-utils`, which allows you to mount CIFS-based shared
-folders inside the VM. For example, on Windows 10 the `C:\Users` folder is shared and only needs locally
+The Minishift [Boot2Docker](https://github.com/minishift/minishift-b2d-iso) as well as the
+Minishift [CentOS](https://github.com/minishift/minishift-centos-iso) ISO include `cifs-utils` and
```

Comment:
ISOs is not an official term, I would qualify them separately (as in the x ISO and the y ISO) to ease the grammar thing :)